packages/babel-plugin-transform-async-to-generator/test/fixtures/async-to-generator/shadowed-promise-import/options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "babelrc": false,
```

Comment:
I'm actually not sure if this works, but the intent is to prevent babel from merging this .babelrc with the one above it 😆